spacy_transformers/tests/util.py

```diff
@@ -132,6 +136,7 @@ def DummyTransformer(
             "tokenizer": DummyTokenizer(),
             "grad_factor": 1.0,
             "flush_cache_chance": 0.0,
+            "transformers_config": {"output_attentions": False}
```

Comment:
```suggestion
            "transformer_config": {"output_attentions": False}
```

Reply:
Is this additional argument actually needed - does it not work with the default empty `dict`?

Reply:
It needs the output_attentions in line 146 where it handles the new output format.
```
    if trf_config["output_attentions"] is True:
```
but have changed that to:
```
if ("output_attentions" in trf_config) and (trf_config["output_attentions"] is True):
```

as I expect there might be some backward compatibility issues with the first approach.